File: resourcespoet/src/test/java/com/commit451/resourcespoet/ValueTest.java
package com.commit451.resourcespoet;

import org.junit.Assert;
import org.junit.Test;

import java.io.File;
import java.util.ArrayList;

/**
 * Tests reading a value
 */
public class ValueTest {

    @Test
    public void valueTest() throws Exception {
        ClassLoader classLoader = getClass().getClassLoader();
        File file = new File(classLoader.getResource("bool.xml").getFile());
        ResourcesPoet poet = ResourcesPoet.create(file);
        String value = poet.value(Type.BOOL, "is_cool");
        Assert.assertEquals("true", value);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/ValueTest.kt
package com.commit451.resourcespoet

import org.junit.Assert
import org.junit.Test

import java.io.File

/**
 * Tests reading a value
 */
class ValueTest {

    @Test
    fun valueTest() {
        val classLoader = javaClass.classLoader
        val file = File(classLoader.getResource("bool.xml")!!.file)
        val poet = ResourcesPoet.create(file)
        val value = poet.value(Type.BOOL, "is_cool")
        Assert.assertEquals("true", value)
    }
}
